src/shogun/features/DenseFeatures.cpp

```diff
@@ -1067,6 +1048,24 @@ CFeatures* CDenseFeatures<ST>::create_merged_copy(CFeatures* other)
 	return create_merged_copy(list);
 }
 
+template <class ST>
+Some<CDenseFeatures<ST>> CDenseFeatures<ST>::view(const SGVector<index_t>& subset)
```

Comment:
the problem with this is that this right away makes it inaccessible/usable from SWIG interfaces... we are having difficulties with exposing Some in SWIG